addons/csi/openstack/controllerplugin.yaml

```diff
@@ -69,11 +78,13 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-provisioner
-          image: '{{ Image "registry.k8s.io/sig-storage/csi-provisioner:v3.1.0" }}'
+          image: '{{ Image "registry.k8s.io/sig-storage/csi-provisioner:v3.4.1" }}'
```

Comment:
this seems to introduce a regression

Upstream has set a topology support as default - https://github.com/kubernetes/cloud-provider-openstack/pull/1430, which breaks environments where the AZs between nova and cinder are mismatched.

Since KKP currently doesn't support any customization of the CSI driver, only way forward is to build custom addons image and hardcode this flag on the controller plugin
```
--feature-gates=Topology=false
``` 

more in: https://github.com/kubernetes/cloud-provider-openstack/issues/2185